pkg/auth/providerrefresh/refresh_daemon.go

```diff
@@ -187,6 +187,11 @@ func (r *refresher) triggerUserRefresh(userName string, force bool) {
 		return
 	}
 
+	if ref == nil {
```

Comment:
I don't know if this change is related to the original issue...if not we should move it to a separate pr.

Reply:
It is not related to the original issue, but the change is needed for testing and associated with rancher/rancher/issues/17958 (@cjellick not sure if fix is sufficient to address issue). This commit can be removed before merging.

Reply:
Fix was already merged, will remove & rebase